README.md

```diff
@@ -1,11 +1,60 @@
+# pygfx
 [![CI](https://github.com/pygfx/pygfx/workflows/CI/badge.svg)](https://github.com/pygfx/pygfx/actions)
 [![Documentation Status](https://readthedocs.org/projects/pygfx/badge/?version=latest)](https://pygfx.readthedocs.io/en/latest/?badge=latest)
 [![PyPI version](https://badge.fury.io/py/pygfx.svg)](https://badge.fury.io/py/pygfx)
 
-# pygfx
+A python render engine targeting Vulkan/Metal/DX12.
+
+
+<p align="center">
+<img src="./docs/_static/readme_sponza.png" alt="drawing" width="200"/>
+<img src="./docs/_static/readme_pbr_example.webp" alt="drawing" width="200"/>
+<img src="./docs/_static/readme_torus_knot_wire.png" alt="drawing" width="200"/><br />
+</p>
+<p align="center">
+(Check out the <a href="https://pygfx.readthedocs.io/en/latest/_gallery/index.html">example gallery</a> for more renders.)
+</p>
+
+## Installation
+
+If you are just starting out, or have no requirement on the canvas use
+
+```bash
+pip install -U pygfx glfw
+```
+
+If you already have a wgpu-compatible canvas (e.g., Qt, pyside, wx) installed
+you may choose to not install glfw. Similarly, if you only wish to use offscreen
+rendering (eg. in CI) you will also not need glfw.
```

Comment:
Since glfw is really light there is not a problem if its installed "extra", so I'd put less emphasis on the not-installing part. Something along the lines of:

```suggestion
The glfw library provides a lightweight window to render to. It's not strictly needed
if you only use pygfx from the notebook, use offscreen rendering, or plan to use 
another wgpu-compatible canvas such as PyQt, PySide, or wx.
```